Cargo.toml

```diff
@@ -100,6 +100,8 @@ flume = "0.11"
 futures-lite = "2"
 getrandom = "0.2"
 glam = "0.29"
+# TODO: Use `half` directly again after <https://github.com/starkat99/half-rs/pull/111> is released upstream.
+half = { package = "half-2", version = "2.4.1" }
```

Comment:
The crate [`half` v1.8.2](https://searchfox.org/mozilla-central/rev/18bfd136df7f2fe618d8105549cccebb75ce92f5/Cargo.lock#2767) is already in Firefox's tree. I'd like to avoid having to bump its version (once that PR lands) since other crates would then need their versions bumped. Looking at the changelong of [`half` v2+](https://github.com/starkat99/half-rs/compare/v1.8.2...v2.4.1), I don't see anything critical that we would need. Another thing to consider is that rust will get an [`f16` type](https://github.com/rust-lang/rust/issues/116909) at some point in the future.

Could we instead use `half` v1.8.2 with a wrapper that implements `Arbitrary`?

Reply:
I'll see if that's possible

Reply:
We've decided in the meeting to go ahead with the newly updated `half` crate, hoping the audit load isn't too heavy.